versioned_docs/version-1.3/apis-tools/go-client/get-started.md

```diff
@@ -214,7 +214,7 @@ Now, we want to do some work within our process. Follow the steps below:
 - Set the **type** of the second task to `fetcher-service`.
 - Set the **type** of the third task to `shipping-service`.
 
-5. Additionally, for the service task `Collect Money` set a [**task-header**](/docs/1.3/components/modeler/bpmn/service-tasks/#task-headers) with the key `method` and the value `VISA`. This header is used as a configuration parameter for the payment-service worker to hand over the payment method.
+5. Additionally, for the service task `Collect Money` set a [**task-header**](../../components/modeler/bpmn/service-tasks/service-tasks.md/#task-headers) with the key `method` and the value `VISA`. This header is used as a configuration parameter for the payment-service worker to hand over the payment method.
```

Comment:
There were a few of these stragglers -- links that were crossing versions, because of the way they were written. We'll want to update each of these for each archived version.

Reply:
In more recent versions of the docs, I've tried to catch and correct these. All of this to say, it should get better.